extensions/amp-dailymotion/0.1/test/test-amp-dailymotion.js

```diff
@@ -37,6 +37,7 @@ describe('amp-dailymotion', () => {
       }
       if (optCustomSettings) {
         dailymotion.setAttribute('data-start', 123);
+        dailymotion.setAttribute('data-param-origin', 'example.com');
```

Comment:
yeah but for the purpose of testing, it doesn't matter, would be nice to have a test that ensure proper encoding.